.github/workflows/update_yarn_lock.yaml

```diff
@@ -0,0 +1,49 @@
+name: Update yarn.lock
+
+on:
+  workflow_call:
+    inputs:
+      pr_repository:
+        description: The repository to which the PR branch should be pushed
+        required: true
+        type: string
```

Comment:
I want to avoid this parameter, and I think I can derive the value from the source repository name, but for now this is easier and I can do the more complex name derviation in a follow-up